doc/tutorials/agl-vms.rst

```diff
@@ -243,25 +199,26 @@ In this demo, we use acrn-kernel as the baseline for development for AGL.
 
       $ cd workspace
       $ git clone https://github.com/projectacrn/acrn-kernel
-      $ git checkout tags/acrn-2018w49.3-140000p
-      $ make menuconfig
+      $ git checkout tags/acrn-2019w39.1-140000p
+      $ cp kernel_config_uos .config
+      $ make olddefconfig
+      $ vi .config
```

Comment:
Shouldn't this `.config` editing step be done *before* running `make olddefconfig`?

Reply:
Yes, you are right, I will make changes